### ISSUE TITLE
Run a skipped test when selected

### ISSUE THREAD

**Issue by JorritSalverda** [xunit/resharper-xunit#103]
When I specifically select a test - even a skipped one - I would expect to be able to run it. With nunit this is possible. Perhaps this isn't the plugins fault but just the difference between xunit and nunit itself.

**Comment by citizenmatt** [comment]
Yep, this is the behaviour of the xunit framework, not the plugin. Your code specifies the test as passing, and xunit always respects this. I don't think there's a way to work around this. @bradwilson?

**Comment by bradwilson** [comment]
https://github.com/xunit/xunit/issues/701
